src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,279 @@
+              <slot
+                name="column-2-social-title"
+                class="o-footer__social-media-title"
+              ></slot>
+              <ul class="o-footer__social-media-list">
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-0"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-1"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-2"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-3"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-4"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-5"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </axa-container>
+      </footer>
+    `;
+  }
+
+  updated() {
+    this._removeEmptyListElements();
+  }
+
+  _removeEmptyListElements() {
+    this.shadowRoot
+      .querySelectorAll('.o-footer__main-content-panel-list-item')
+      .forEach(el => {
+        const label = el.querySelector('slot').assignedNodes()[0];
+        // Second part of IF-statement is an IE11 workaround
+        if (AXAFooter._listElementHasNoContent(label)) {
+          el.style.display = 'none';
+        }
+      });
+  }
+
+  _handleAccordionClick = index => {
+    this._accordionActiveIndex =
+      index === this._accordionActiveIndex ? -1 : index;
+    this.requestUpdate();
+  };
+
+  _handleLinkClick = ev => {
+    if (this.dynamic) {
+      ev.preventDefault();
+
+      const href = AXAFooter._extractNestedHref(ev);
+
+      if (href) {
+        this.onItemClick(href);
+        this.dispatchEvent(
+          new CustomEvent('axa-footer-click', {
+            detail: href,
+            bubbles: true,
+            cancelable: true,
+          })
+        );
+      }
+    }
+  };
+
+  static _listElementHasNoContent(label) {
+    return !label || label.toString() === '[object Text]';
+  }
+
+  static _extractNestedHref(ev) {
+    let element;
+    if (!ev.target || !ev.target.href) {
```

Comment:
Please use deconstruction. Doing `ev.target` twice is duplication of code. Is like in JAVA terms calling the getter method of a class twice instead of storing it locally.

Something like so:

```js
static _extractNestedHref({target}) {
    let element;
    if (!target || !target.href) {
       ...
    }
```